ChargerMail/src/User.java

```diff
@@ -0,0 +1,87 @@
+
+/**
+ * Class to implement a user within email client system
+ * @author tevonwalker
+ *
+ */
+
+public class User {
+	
+	/**
+	 * Constructs user object and sets username to username. Also sets up the remote and local sites
+	 * @param username name to use for new user
+	 */
+	
+	public User(String username){
+		this.username = username;
+		localSite = new Site();
+		remoteSite = new Site();
+	}
+	
+	/**
+	 * Set user name 
```

Comment:
Do you want to have javadoc stuff for everything? Dr. Li said that getters, setters, and constructors didn't need it; only methods that did something important and the class. But we can do everything if you think that's better.